### ISSUE TITLE
bug with EDD recurring payment

### ISSUE THREAD

**Issue by Kurosagi44** [samikeijonen/edd-members#19]
1) with the option "check to enable membership creation" enabled
When customers purchase a subscriptions (for example at 43.90/month), the first payment is correct (43.90 in my example), but the subscription created is made with a wrong amount: (52.04/month or sometimes 42.70). The problem is that this amount is sent to Stripe for the subscription plan creation, which is bad for the business (people expect to pay 43.90, but they are charged another amount).

2) with the option "check to enable membership creation" disabled
All works properly, seems there is no bug.

**Comment by samikeijonen** [comment]
I'm not sure I fully understand the bug. In short, is your subscriptions amount wrong in Stripe?

EDD Members should not interfere for payments at all, it should only restrict content by adding date value in user meta.

**Comment by Kurosagi44** [comment]
Yes, in short, that's right: subscription amount is wrong on Stripe AND on
EDD Subscription.

I know EDD members should not interfere with payment, but according to EDD
support team, that bug appears only when the EDD Member checkbox is
enabled. Here are a screenshot of the bug.

However, maybe the EDD guy was wrong and maybe it is due to their plugin.
All I know is I have all plugins up to date (EDD+recurring+members).

[image: Images intégrées 1]

2017-10-13 18:44 GMT+02:00 Sami Keijonen <notifications@github.com>:

**Comment by samikeijonen** [comment]
Can you try to sent image again, I don't seem to see it.
